src/components/GoodsList/GoodsList.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import './GoodsList.css';
+import PropTypes from 'prop-types';
+
+const selectItems = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
```

Comment:
For this task, it's ok, but better not to hardcode values and use loops, or array methods for filling the arrays with some values